ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -923,7 +922,7 @@ def main():
             cmd=dict(required=True, choices=['create', 'bind', 'renumber', 'unbind', 'destroy', "connect-vms", "disconnect-vms"]),
             vm_set_name=dict(required=False, type='str'),
             topo=dict(required=False, type='dict'),
-            vm_names=dict(required=True, type='list'),
+            vm_names=dict(required=False, type='list'),
```

Comment:
I think @neethajohn asked for a similar change in another PR. please don't allow skip this option but allow provide empty list.